src/Ocelot/DependencyInjection/ConfigurationBuilderExtensions.cs

```diff
@@ -1,24 +1,25 @@
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
-using System.Text.RegularExpressions;
-
-using Ocelot.Configuration.File;
-
-using Microsoft.AspNetCore.Hosting;
-using Microsoft.Extensions.Configuration;
-using Microsoft.Extensions.Configuration.Memory;
-
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.Memory;
 using Newtonsoft.Json;
+using Ocelot.Configuration.File;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
 
 namespace Ocelot.DependencyInjection
 {
     public static class ConfigurationBuilderExtensions
     {
+        private static readonly Regex _reg = new Regex(@"^ocelot\.(.*?)\.json$", RegexOptions.IgnoreCase | RegexOptions.Singleline | RegexOptions.Compiled, TimeSpan.FromMilliseconds(100));
+
         [Obsolete("Please set BaseUrl in ocelot.json GlobalConfiguration.BaseUrl")]
         public static IConfigurationBuilder AddOcelotBaseUrl(this IConfigurationBuilder builder, string baseUrl)
         {
+            Regex.CacheSize += 100;
```

Comment:
What is the default, current value of the cache in runtime?
So, why did you increase this property value?

Reply:
`AddOcelotBaseUrl` is `Obsolete`, so this is wrong location to handle the work.
Should be removed!

Fixed in https://github.com/ThreeMammals/Ocelot/pull/1348/commits/364d1b63ee39370f53c5e96bce2395d92b068be5 ✔️